tensorflow/core/kernels/tile_functor.h

```diff
@@ -29,13 +30,13 @@ namespace internal {
 template <typename Device, typename T>
 void TileSimple(const Device& d, Tensor* out, const Tensor& in);
 
-template <typename Device, typename T, int NDIM>
+template <typename Device, typename Tmultiples, typename T, int NDIM>
```

Comment:
Minor nit: it seems like T should come before Tmultiples since we usually put the 'main datatype' first and then the auxilliary one.

Reply:
Thanks @vrv, the order of the typename has been rearranged.